package/babel/preset.js

```diff
@@ -27,18 +28,28 @@ module.exports = function config(api) {
           loose: true,
           exclude: ['transform-typeof-symbol']
         }
+      ],
+      moduleExists('@babel/preset-typescript') && [
+        '@babel/preset-typescript',
+        { allExtensions: true, isTSX: true }
+      ],
+      moduleExists('@babel/preset-react') && [
+        '@babel/preset-react',
+        {
+          development: isDevelopmentEnv || isTestEnv,
+          useBuiltIns: true
+        }
       ]
     ].filter(Boolean),
     plugins: [
       'babel-plugin-macros',
-      [
-        '@babel/plugin-proposal-class-properties',
-        { loose: true }
-      ],
-      [
-        '@babel/plugin-transform-runtime',
-        { helpers: false }
-      ]
+      ['@babel/plugin-proposal-class-properties', { loose: true }],
```

Comment:
it would also be nice to document how developer can customize this babel preset

https://github.com/rails/webpacker/issues/2775

Reply:
Should be easy, I think one can import and extend the base config just like webpack or just copy over and customise as needed.